src/modules/shared/pages/switch-context/switch-context.component.ts

```diff
@@ -74,7 +74,7 @@ export class PageSwitchContextComponent  extends CoreComponent implements OnInit
       const currentOrgUnitId = this.authenticationStore.getUserContextInfo().organisation?.organisationUnit.id;
 
       this.authenticationStore.updateSelectedUserContext({
-        type: userInfo.type,
+        type: userInfo.roles[0].role,
```

Comment:
This will not work when user has more than one type, add a TODO here

Reply:
This is the next task. this one was just to not break what we currently have